book/getting-started/run-a-wallaroo-application-docker.md

```diff
@@ -0,0 +1,201 @@
+# Run a Wallaroo Application in Docker
+
+In this section, we're going to run an example Wallaroo application in Docker. By the time you are finished, you'll have validated that your Docker environment is set up and working correctly. If you haven't already completed the Docker setup [instructions](book/getting-started/docker-setup.md), please do so before continuing.
+
+There's a couple Wallaroo support applications that you'll be interacting with for the first time:
+
+- Our Metrics UI that allows you to monitor the performance and health of your applications.
+- Giles receiver is designed to capture TCP output from Wallaroo applications.
+- Giles sender is used to send test data into Wallaroo applications over TCP.
+- Machida, our program for running Wallaroo Python applications.
+
+You're going to set up our "Celsius to Fahrenheit" example application. Giles sender will be used to pump data into the application. Giles receiver will receive the output and our Metrics UI will be running so you can observe the overall performance.
+
+The Metrics UI process will be run in the background. The other three processes (receiver, sender, and Wallaroo) will run in the foreground. We recommend that you run each process in a separate terminal.
+
+NOTE: You might need to run Docker commands with sudo depending on how you set up Docker.
+
+Let's get started!
+
+## Terminal 1, Start the Wallaroo Docker container
+
+```bash
+docker run --rm -it --privileged -p 4000:4000 -v /tmp/wallaroo-docker/wallaroo-src:/src/wallaroo -v /tmp/wallaroo-docker/python-virtualenv:/src/python-virtualenv --name wally wallaroo-labs-docker-wallaroolabs.bintray.io/dev/wallaroo:7b1f3550
+```
+
+### Breaking down the Docker command
+
+- `docker run`: The Docker command that will start a new container.
+
+- `--rm`: This flag will automatically clean up the container and remove the file system on exit.
+
+- `-it`: This flag will allow us to work with interactive processes by allocating a tty for the container.
+
+- `--privileged`: This flag gives the container access to the hosts' devices. This allows us to take advantage of certain system calls used by Wallaroo, specifically `mbind` and `set_mempolicy`. This flag is optional, but by excluding it there will be a performance degradation in Wallaroo's processing capabilities.
```

Comment:
"this flag gives" -> "Give the ..."
"This allows us..." -> "This allows certain system calls to be used by ..."
"This flag is optional" -> "This setting is optional, but omitting it will result in a performance degradation."